README.md

```diff
@@ -188,7 +198,13 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [AniTrend](https://play.google.com/store/apps/details?id=com.mxt.anitrend)
 * [LiveChart](https://play.google.com/store/apps/details?id=me.livechart.android)
 * [Crunchyroll](https://play.google.com/store/apps/details?id=com.crunchyroll.crunchyroid)
-* [Animity](https://github.com/kl3jvi/animity)
+* [AnimeNow](https://github.com/AnimeNow-Team/AnimeNow)
+
+#### Android TV
```

Comment:
why android TV? Most seem like mobile apps

Reply:
There are separate versions for tv(Saikou tv , kuro tv) , some of them can be used on the tv with the same apk which is used on Android.